src/components/Instructor/Instructor.component.jsx

```diff
@@ -0,0 +1,214 @@
+import "./Instructor.scss";
+
+import React, { useState, useEffect } from "react";
+import { Link } from "react-router-dom";
+import { Table, Tag, Empty, Avatar, Tooltip, message } from "antd";
+import ErrorPage from "../../common/ErrorPage/ErrorPage.component";
+import Loading from "../../common/Loading/Loading";
+import PaginationCommonContainer from "../../common/Pagination/PaginationCommon";
+import { InputSearchCommon } from "../../common/Input/Input";
+import useDebounce from "../../common/useDebounce/useDebounce.component";
+import api from "../../api/index.api";
+
+const pageSize = 10;
+
+const InstructorComponent = ({ GetAllInstructor, token }) => {
+	const [instructors, setInstructors] = useState([]);
+	const [err, setError] = useState(null);
+	const [loading, setLoading] = useState(false);
+	const [currentPage, setcurrentPage] = useState(1);
+	const [total, setTotal] = useState(0);
+	const [search, setSearch] = useState("");
+	const debouncedKeysearch = useDebounce(search.trim(), 300);
+
+	const fSuccess = (data) => {
+		const { instructors, total } = data;
+		setTotal(total);
+		setLoading(false);
+		setInstructors(instructors);
+	};
+
+	const fFailure = (err) => {
+		setError(err);
+	};
+
+	useEffect(() => {
+		if (debouncedKeysearch || debouncedKeysearch === "") {
+			GetAllInstructor(
+				{ limit: pageSize, offset: currentPage, search: debouncedKeysearch },
+				fSuccess,
+				fFailure,
+			);
+		}
+	}, [GetAllInstructor, currentPage, search, debouncedKeysearch]);
```

Comment:
Khi change keyword thì cho currentPage trở về 1

```suggestion
	useEffect(() => {
	               setCurrentPage(1)
			GetAllInstructor(
				{ limit: pageSize, offset: 1, search: debouncedKeysearch },
				fSuccess,
				fFailure,
			);
		}
	}, [GetAllInstructor, search, debouncedKeysearch]);
```